EnglishFilter/sections/specific.txt

```diff
@@ -143,6 +143,7 @@ kiss-anime.ws##script[id^="BB_SLOT_"] ~ div[id][style^="overflow: hidden; width:
 ! common names
 aplayer2.me,viralizeit.net,sex3.com,softonic.com,3pornstarmovies.com,flvto.biz,govtsmartjob.com,cutearn.ca,dfporn.net,cll.press,hellporno.net,youramateurporn.com,getlink.pw,embedtvseries.com,keygames.com,uclick.in,flv2mp3.by,doodhwali.com,smutindia.com,dvdcover.com,xxxadultphoto.com,gabicoins.com,qpdownload.com,windows10portal.com,2conv.com,javcloud.com,underhentai.net,cutwin.com,hqasiananal.com,bradsknutson.com,flv2mp3.org,subscene.com,yourlink.in,iplayer.pw,qpornx.com,sofifa.com,jawcloud.co,indiansgetfucked.com,medi-bayreuth.de,oxforddictionaries.com,tube18.sex##.banner
 !
+appledaily.com#$#.ndAritcle_headPic,.mediabox,.ndArticle_margin { visibility: visible !important; }
```

Comment:
please don't add comma-separated selectors. Better try to fix it in another way

Reply:
also there are missed social share buttons

Reply:
AdamWr said paywall issues belong to AdGuard Annoyances filter, not AdGuard Base filter.